api/layer/object.go

```diff
@@ -262,7 +272,7 @@ func (n *layer) PutObject(ctx context.Context, p *PutObjectParams) (*data.Extend
 	n.log.Debug("put object",
 		zap.String("reqId", reqInfo.RequestID),
 		zap.String("bucket", p.BktInfo.Name), zap.Stringer("cid", p.BktInfo.CID),
-		zap.String("object", p.Object), zap.Stringer("oid", id))
+		zap.String("object", p.Object), zap.Stringer("oid", id), zap.Int64("size", p.Size))
 
 	newVersion.OID = id
 	newVersion.ETag = hex.EncodeToString(hash)
```

Comment:
Ah, looks like you still have `treeService.AddVersion` for the object below. This explains epoch-bound degradation. Can we drop it?